docs/dev/HadoopWinutils.md

```diff
@@ -0,0 +1,11 @@
+## Hadoop Winutils
+
+Hadoop Winutils native libraries are required to run Drill on Windows. The last version presented in maven repository is 2.7.1 and doesn't updated anymore.
+So instead of heaving a dependency Drill now contains required files as resources at distribution/src/main/resources.
```

Comment:
That's why Winutils version matching Hadoop version used in Drill is located in distribution/src/main/resources.

Reply:
Fixed, thanks.